build.sbt

```diff
@@ -106,6 +119,18 @@ lazy val `anorm-akka` = (project in file("akka"))
       "com.typesafe.akka" %% "akka-stream-contrib" % "0.6" % Test)
   )).dependsOn(anorm)
 
+lazy val pgVer = sys.props.get("postgres.version").getOrElse("9.4.1212")
+lazy val jsonVer = sys.props.get("playJson.version").getOrElse("2.6.0-M3")
```

Comment:
Do you need any of the new features of play-json? This might cause binary compatibility issues if you're using anorm with Play 2.5.x.